### PR TITLE
Sync for companyTeam and companyteamrole.

### PR DESCRIPTION
**Pull request by mkaundal-topleft**

### CODE REVIEW

**Comment by mkaundal-topleft** [comment]
Made changes for failing test cases, but still I am getting some failing test cases. If the reviewer can look onto those and point of the mistake I am making.

File: djconnectwise/models.py [hunk 1]
```diff
@@ -446,6 +446,42 @@ def __str__(self):
...
+    team_role = models.ForeignKey('CompanyTeamRole',
+                                  blank=True, null=True,
+                                  on_delete=models.CASCADE)
+    location = models.ForeignKey('Location', blank=True,
+                                 null=True, on_delete=models.CASCADE)
+    contact = models.ForeignKey('Contact', blank=True,
+                                null=True, on_delete=models.CASCADE)
+    member = models.ForeignKey('Member', blank=True,
+                               null=True, on_delete=models.CASCADE)
+    accountManagerFlag = models.BooleanField()
```
Comment (kti-cameron) [inline djconnectwise/models.py:461]: Lets stick to snake case. Can you change this to `account_manager_flag` instead? And any other cases where camel case is used on the new models?

**Comment by kti-cameron** [comment]
Just that one change so far. I still need to look a bit closer into why those tests are failing.

**Comment by kti-cameron** [comment]
Thanks for the changes. I think I've got those test issues resolved now.

**Pull request merged**